docs/tutorials/morpheusvm/intro.md

```diff
@@ -0,0 +1,818 @@
+# Introduction
+
+In this tutorial, we will go through how to build MorpheusVM, a HyperVM which
+supports token transfers. Although an implementation of MorpheusVM already
+exists, we will set out to build our own version from scratch. The outline of
+this tutorial is as follows:
+
+- Prerequisites
+- Initializing MorpheusVM
+- Creating Transfer, Pt. 1
+- Implementing Storage
+  - Read/write functions
+  - StateManager
+- Creating Transfer, Pt. 2
+- Bringing Everything Together
+- Options
+- Workload Tests
+
+This introduction will go over everything up to and including the section
+"Bringing Everything Together".
+
+## Prerequisites
+
+- You are using Go 1.21.12
+
+To confirm, you can run:
+
+```sh
+go version
+```
+
+And you should see the ouptut (slightly different depending on your machine):
+
+```
+go version go1.21.12 darwin/arm64
+```
+
+## Cloning HyperSDK
+
+To get started, clone the HyperSDK:
+
+```sh
+git clone git@github.com:ava-labs/hypersdk.git
+cd hypersdk
+```
+
+## Initializing MorpheusVM
+
+To start, we'll create a new directory in examples and initialize the go mod:
+
+```sh
+mkdir examples/tutorial
+cd examples/tutorial
+go mod init
+```
+
+We now define our VM constants. We'll create a new package to store these
+values:
+
+```sh
+mkdir consts
+cd consts/
+```
+
```

Comment:
Can we skip moving into the different sub-directories? We should stay at the root of our directory and people can open up the correct file.